nodes/Soroban/actions/server/findPayments/helpers/helpers.ts

```diff
@@ -0,0 +1,101 @@
+import axios from 'axios';
+import Payment from '../../../entities/Payment';
+import { Server } from 'soroban-client';
+import NoPaymentFoundError from '../../../../../../common/errors/stellar/NoPaymentFoundError';
+import { IPaymentRecord } from '../../../../../../common/interfaces/stellar/IPaymentRecord';
+import IAsset from '../../../../../../common/interfaces/stellar/IAsset';
+
+enum FiltersTypeEnum {
+	LIMIT = 'limit',
+	ORDER = 'order',
+}
+
+export async function getPayments(
+	server: Server,
+	destinationAccount: string,
+	limit: number,
+	order: orderType,
+) {
+	// const firstPayments: Payment[] = [];
+	const url = `${server.serverURL}accounts/`;
+
+	const queryParams = new URLSearchParams();
+	if (limit) queryParams.append(FiltersTypeEnum.LIMIT, limit.toString());
+	if (order) queryParams.append(FiltersTypeEnum.ORDER, order);
+
+	const queryString = `${queryParams.toString()}`;
```

Comment:
I think you can do this
```suggestion
	const queryString = queryParams.toString();
```
Let me know if this works.